src/gui/tray/ActivityItemContent.qml

```diff
@@ -118,32 +114,26 @@ Item {
     ColumnLayout {
         id: activityContentLayout
 
-        anchors.left: thumbnailItem.right
-        anchors.right: parent.right
-        anchors.top: parent.top
-        anchors.bottom: parent.bottom
+        Layout.fillWidth: true
+        Layout.fillHeight: true
 
         spacing: Style.smallSpacing
 
         RowLayout {
             Layout.fillWidth: true
-            Layout.maximumWidth: activityContentLayout.width
 
             spacing: Style.trayHorizontalMargin
 
             EnforcedPlainTextLabel {
                 id: activityTextTitle
                 text: (root.activityData.type === "Activity" || root.activityData.type === "Notification") ? root.activityData.subject : root.activityData.message
-                height: (text === "") ? 0 : implicitHeight
 
-                Layout.maximumWidth: activityContentLayout.width - Style.trayHorizontalMargin -
-                                     (activityTextDateTime.visible ? activityTextDateTime.width + Style.trayHorizontalMargin : 0) -
-                                     (dismissActionButton.visible ? dismissActionButton.width + Style.trayHorizontalMargin : 0)
-                Layout.alignment: Qt.AlignTop | Qt.AlignLeft
+                Layout.fillWidth: true
+                Layout.alignment: Qt.AlignVCenter | Qt.AlignLeft
 
                 elide: Text.ElideRight
                 wrapMode: Text.Wrap
-                maximumLineCount: 1
+                maximumLineCount: 2
```

Comment:
do we often get title that long ?

Reply:
Most of the time, yes -- a lot of talk messages and file changes within the same folder become unintelligible if we restrict them to oneline